### PR TITLE
chore: add release script and update README.md

### LINKED ISSUE #11: As a developer, I can release with semantic versioning
### Technical Requirements

- [x] Use [standard-version](https://github.com/conventional-changelog/standard-version)

### PR DESCRIPTION
**Pull request by remarkablemark**
Closes #11

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,15 @@
 # vinch
 
-An idea creation app.
+[![Build Status](https://travis-ci.org/getvinch/vinch.svg?branch=master)](https://travis-ci.org/getvinch/vinch)
+[![Coverage Status](https://coveralls.io/repos/github/getvinch/vinch/badge.svg?branch=ci-travis)](https://coveralls.io/github/getvinch/vinch?branch=ci-travis)
+[![Dependency status](https://david-dm.org/getvinch/vinch.svg)](https://david-dm.org/getvinch/vinch)
+
+Vinch is an app for ideation.
```
Comment (benox3) [inline README.md]: Nitpick: Should we keep this lowercase to keep it consistent?

Comment (remarkablemark) [inline README.md]: Updated description

File: package.json [hunk 2]
```diff
@@ -28,7 +29,8 @@
     "eslint-plugin-prettier": "^3.1.1",
     "husky": "^3.0.8",
     "lint-staged": "^9.4.2",
-    "prettier": "^1.18.2"
+    "prettier": "^1.18.2",
+    "standard-version": "^6"
```
Comment (remarkablemark) [inline package.json]: v7 follows npm, which only bumps by minor if it has `BREAKING CHANGE`, which is not what we want

File: README.md (hunk 1 above)
Comment (remarkablemark) [reply README.md]: Going with capitalization as it has a more business feel

**Pull request merged**